README.md

```diff
@@ -27,8 +27,8 @@ need for a database class library to hide the differences between the
 different databases (encapsulate the differences) so we can easily
 switch databases.
 
-The library currently supports MySQL, Interbase, Sybase, PostgreSQL, Oracle,
-Microsoft SQL server,  Foxpro ODBC, Access ODBC, Informix, DB2,
+The library currently supports MySQL, Firebird & Interbase, PostgreSQL, SQLite3, Oracle,
+Microsoft SQL Server, Foxpro ODBC, Access ODBC, Informix, DB2, Sybase,
```

Comment:
Sybase is already listed on the line below

Reply:
I just reordered them a bit, first open source, then the others.

Reply:
I would vote for a simple alphabetical ordered list

Reply:
Well, I primarly just wanted to add the 2 unlisted db types and fix a typo with this line changes. The sorting listing first open source and second a little bit by popularity is just low priority. Do as you want.

Reply:
I think we should pull FoxPro out , as it was end-of-life 8 years ago, and we aren't supporting the driver anymore